metrics/coverage_high_water_mark

```diff
@@ -1 +1 @@
-93.7200
\ No newline at end of file
+93.4700
```

Comment:
For some reason the unit tests found more source files this time.  Frustrating, as we told nosetests to include everything...